docs/reference_guides/model_libraries/generic/costing/process_costing.rst

```diff
@@ -0,0 +1,891 @@
+Process Costing Library
+=======================
+
+.. contents:: Contents 
+    :depth: 4
+
+Introduction
+------------
+
+.. rubric:: IDAES Process Costing Framework
+
+This document outlines an example process costing library to support IDAES unit models. For a general overview of costing capabilities in IDAES, see the :ref:`IDAES Process Costing Framework<reference_guides/core/costing/costing_framework:IDAES Process Costing Framework>`.
+
+.. note:: Process costing methods are available for most of the unit operations in the IDAES Unit Model Library (Compressor, CSTR, Flash, Heater, HeatExchanger, HeatExchangerNTU, PFR, PressureChanger, Pump, StoichiometricReactor, Turbine, and generic vessels).
+
+The process costing module outlines capital cost methodology for generic IDAES unit operations assuming a 2018 cost basis.
+
+Table 1. Main Variables added to the unit block ("self.costing").
+
+=========================== ============================== ======== ==========================================================================
+Variable                    Symbol                         Units    Notes
+=========================== ============================== ======== ==========================================================================
+Purchase cost               :math:`purchase\_cost`         dollars  Purchase cost
+Base cost per unit          :math:`base\_cost\_per\_unit`  unitless Base cost per unit
+Base cost                   :math:`base\_cost`             unitless Base cost (base cost per unit * number of units)
+Number of units             :math:`number\_of\_units`      unitless Number of units to be costed (to take advantage of the economics of scale)
+=========================== ============================== ======== ==========================================================================
+
+.. note:: number of units by default is fixed to 1 and the user must unfix this variable to optimize the number of units. Also, `number of units` can be built as a continuous variable or an integer variable. If the latter, the user must provide an mip solver. Use the global costing argument for this purpose (integer_n_units=True or False).
+
+Available IDAES Process Costing Module Methods
+----------------------------------------------
+
+A default costing module has been developed primarily based on base cost and purchase cost correlations 
```

Comment:
Do not call this the default method.

I would move this and everything before table 2 to the introduction (and expand upon it).